python/cuml/dask/common/__init__.py

```diff
@@ -14,12 +14,12 @@
 # limitations under the License.
 #
 
-from cuml.dask.common.comms import CommsContext, worker_state, default_comms
+from cuml.dask.common.comms import CommsContext, worker_state
```

Comment:
This important should be able to go away now that this functionality is inside RAFT